misty.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.rdoc_options = ['--charset=UTF-8']
   spec.extra_rdoc_files = %w[README.md LICENSE.md]
 
-  spec.add_dependency 'json', '~> 2.0'
+  spec.add_dependency 'json', '>= 1.8.3', '< 3'
```

Comment:
Rails version is 4.2.4 if you are intereseted you can take a look [here](https://github.com/sapcc/elektra)